src/main/java/org/broad/igv/ui/panel/ZoomSliderPanel.java

```diff
@@ -193,9 +200,33 @@ protected void paintHorizontal(Graphics g) {
                 //g.drawImage(slider, x + 1, y, null);
             }
         }
+
+        paintVisibilityThresholds(transGraphics);
         transGraphics.dispose();
     }
 
+    //Adds an indicator of which zoom level will display reads/features
+    private void paintVisibilityThresholds(final Graphics2D transGraphics) {
+        if(numZoomLevels > 1) {
+            List<Integer> visibilityThresholds = IGV.getInstance().getAllTracks().stream()
+                    .map(Track::getVisibilityWindow)
+                    .filter(i -> i > 0)
+                    .sorted()
+                    .distinct()
+                    .map(threshold -> this.getReferenceFrame().calculateZoom(0, threshold))
+                    .collect(Collectors.toList());
+
+            transGraphics.setColor(TRANSPARENT_BLUE);
+            Rectangle maxZoom = zoomLevelRects[zoomLevelRects.length - 1];
+            for (Integer window : visibilityThresholds) {
```

Comment:
There's a bug here somewhere,  array index can be negative.   I'll drop more details in the "conversation" widget